qt/app.py

```diff
@@ -1182,11 +1182,26 @@ def updateTimeLine(self, refreshSnapshotsList = True):
                 item = self.timeLine.addSnapshot(sid)
             self.timeLine.checkSelection()
 
+    def validateOnTakeSnapshot(self):
+        hasMissing, missing = snapshots.hasMissing(self.config.include())
+        if hasMissing:
+            msgMissing = '\n'.join(missing)
+            msg = f'{_("The following folders are missing")}:\n\n{msgMissing}\n\n{_("Do you want to proceed?")}'
+            answer = messagebox.warningYesNo(self, msg)
```

Comment:
The question is if this need to be a warning or can it be just an info? I am not sure myself.
If you want to keep it a warning then I would suggest to create a new `messagebox.py::warning()` like `messagebox.py::info()`. Technically it is not more then a message dialog with an OK button and a warning symbol.

Reply:
I am not sure about this comment.

I am using the `warningYesNo` to double-check with the user if we should take the snapshot

![image](https://github.com/bit-team/backintime/assets/1108973/8bdf6e5b-746b-4095-ab2e-f7f0b5ceb550)

We could:
* Confirm w/ yes or no (like it is now)
* Show a warning and always take a snapshot (but then change the message, replacing or removing the "Do you want to proceed?"

Both seems good for me. The yes or no is good because the user could fix this before the snapshot is done (it seems my favorite)

Do you have a preference, @buhtz ?

Reply:
Yea, I was wrong. Of course there need to be a yes-no option.